docker/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+TARGET=$1
+done=0
+trap 'done=1' TERM INT
+cd /mapproxy
+
+groupadd mapproxy && \
```

Comment:
Very minor and non-blocking: This will probably lead to some warning message that the group already exists, when restarting an existing container.

Reply:
right it does, can be fixed in follow up pr